pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -54,6 +55,7 @@ type KubeUtil struct {
 
 	kubeletHost              string // resolved hostname or IPAddress
 	kubeletApiEndpoint       string // ${SCHEME}://${kubeletHost}:${PORT}
+	kubeletProxiedEndpoint   bool
```

Comment:
Would expect the field name to reflect the boolean nature of the property - maybe `kubeletProxyEnabled`?

Reply:
sounds good